### PR TITLE
add multilabel RAI vision example for fridge multilabel dataset

### PR DESCRIPTION
**Pull request by imatiach-msft**
## Description

Add multilabel RAI vision example for fridge multilabel dataset.
This includes:
1.) Adding the multilabel fridge dataset to the test app
2.) Updating image explorer view, table view and data characteristics to show the proper labels as an array and whether they are a success or error instance
3.) Some minor UI tweaks for label to be left aligned and green/red circle to be limited within the bounds of the text to fix existing bug.

Screenshot of multilabel example:

image

## Checklist

- [x] I have added screenshots above for all UI changes.
- [x] I have added e2e tests for all UI changes.
- [x] Documentation was updated if it was needed.

### CODE REVIEW

File: apps/dashboard/src/model-assessment-vision/__mock_data__/fridgeMultilabelImages.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const fridgeMultilabelImages = [
```
Comment (romanlutz) [inline apps/dashboard/src/model-assessment-vision/__mock_data__/fridgeMultilabelImages.ts:4]: What is this? Encoded images?

File: libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx [hunk 2]
```diff
@@ -133,18 +137,16 @@ export class ImageList extends React.Component<
...
-              item?.predictedY === item?.trueY
+              predictedY === trueY
                 ? classNames.successIndicator
                 : classNames.errorIndicator
             }
             style={{
-              left: ImagePadding,
-              maxWidth: this.props.imageDim
+              left: ImagePadding
+              // maxWidth: this.props.imageDim
```
Comment (romanlutz) [inline libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx:146]: ? Remove ?

**Review by romanlutz — APPROVED** [review]

**Review by vinuthakaranth — APPROVED** [review]

File: libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx (hunk 2 above)
Comment (imatiach-msft) [reply libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx:146]: removed!

File: apps/dashboard/src/model-assessment-vision/__mock_data__/fridgeMultilabelImages.ts (hunk 1 above)
Comment (imatiach-msft) [reply apps/dashboard/src/model-assessment-vision/__mock_data__/fridgeMultilabelImages.ts:4]: yep base64 encoded images as strings

**Pull request merged**